tests/visco_plastic_yield_max.prm

```diff
@@ -0,0 +1,111 @@
+# Global parameters
```

Comment:
Can you  add a few brief sentences at the top of the parameter file describing what this parameter file and testing and what the expected output is? Effectively what you outlined in your earlier comment?

Reply:
This is done.